Zend/tests/cwd_is_slash_at.phpt

```diff
@@ -18,7 +18,7 @@ if (setlocale(LC_CTYPE, 'Japanese_Japan.932') === false) {
 // Create "表/ン/機能.php"
 shell_exec(
     mb_convert_encoding(
-        'chcp 932 & mkdir "表/ン" & echo ^<?php echo __FILE__, PHP_EOL, dirname(__FILE__);> "表/ン/機能.php"',
+        'mkdir "表/ン" & echo ^<?php echo __FILE__, PHP_EOL, dirname(__FILE__);> "表/ン/機能.php"',
         'CP932',
```

Comment:
Absolute path should be always used in the test for working with files. So here it should be `dirname(__FILE__) . "/表/ン/機能.php"`. Tests can run in different locations and accidentially CWD could be changed, so that could end up with a test fail.